### DISCUSSION TITLE
Akash Accelerate 2024

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#499] [Governance Proposals]
# Overview
With the growth of Akash over the course of 2023 and into 2024, the launch of Akash’s GPU marketplace, and the fervor within the AI space, now is the perfect time to strike while the iron is hot and host the first ever Akash Summit. We’re calling it **Akash Accelerate**. This event will focus entirely on the Akash ecosystem and other key projects leading the decentralized AI space. 

The below information lays out the who, what, when and where of this proposed event. 

# Why host Akash Accelerate?
With the proliferation of AI, now is the perfect time to bring additional recognition to Akash with Akash Accelerate, a day-long summit focused on the Akash ecosystem. This is an opportunity to highlight the people, projects, and technical achievements within and surrounding Akash.

# Projected Outcomes
Here are the metrics we plan to track to gauge the performance of the event:

- Number of RSVPs
- Number of attendees
- Number of press attendees
- Number of leads generated
- Attendee survey feedback
- Social media mentions
- Livestream views
- Recording views
- Website traffic
- New Akash Insider applicants and onboarded

There are also intangible benefits that come from hosting events: 
- Relationships generated
- Attention brought to Akash
- Goodwill among community members 
- Thought leadership
- Word of mouth

# Timeline

1. Github Discussion: Open for one week review and feedback
2. Gov Prop LIVE: Mid March
3. Confirm Event Management Company : Mid March
4. Confirm Content and Run of show : Mid April
5. Being Promotion: Mid April
6. Host event: End of May, before Consensus (exact date TBD)

# Budget

This being the first time Akash will host an event of this size and scope, I'd like to make it a resounding success. It will include a VIP kickoff dinner, the main event and a conclusion cocktail party. The below budget is based off of my 10+ years of experience running similar events. All dollar amounts below are in USD but will come out of the community pool in AKT.

Kickoff Dinner: $10,000
Networking Happy Hour: $15,000
Venue: $15,000
Catering: $15,000
Equipment Rental: $15,000
Event Management: $15,000
A/V Rental: $5,000
Videographer: $7,500
Swag Items: $15,000
Photographer: $2,500
Hotels: $25,500
Flights: $17,000
Overage: $20,000

**Total Estimated Budget: $177,500** 

**Note:** All unused funds will go back into the community pool

# Final Thoughts 
I forsee Akash Accelerate becoming the Akash Network's annual flagship event. Bringing in VIP's, Journalists, Partners, Developers, Community members, and Akash advocates, I've very excited for the potential of this event. Looking forward to your thoughts and feedback.

**Comment by Zblocker64** [comment]
I think this is a great idea! As someone who attended and represented Akash at ETH Denver in 2024, I see the value in hosting this event. When people get to see how Akash works and how easy it is to deploy, it changes how they think about Akash and makes them want to start using it or start providing to the network. 

I also believe this budget is very much realistic for pulling off an event like this. With how the community pool has been growing, I see nothing wrong in spending AKT this way and believe that the leads and connections created from this event will pay its self back from network usage. 

My one question is will this be an open event? This might be a dumb questions but the proposal says it measures success by number of RSVP, does this mean RSVP are sent out or that anyone can attend and RSVP? Maybe update the proposal with the answer. 

As long as my question is answered, I will be voting YES when this prop hits the chain and look forward to attending Akash Accelerate!

**Comment by CaptainSplince** [comment]
Thanks for the feedback @Zblocker64 . To answer your question:

_My one question is will this be an open event? This might be a dumb questions but the proposal says it measures success by number of RSVP, does this mean RSVP are sent out or that anyone can attend and RSVP? Maybe update the proposal with the answer._

>> This will be an open event. Anyone can attend. Counting the number of RSVPs is just one metric we'll use to decide on the size and scope of future events and to gain a better understanding for the demand of the event. But yes, anyone can attend!

**Reply by okhaimie-dev** [reply]
I love this initiative. I learnt a lot more about Akash from the recently ended in-person event at Denver, and I also see this as an opportunity to foster a stronger bond within the community, so I am in full support of this.

My only question us:
>>> It will include a VIP kickoff dinner

What does this mean from a community POV / how will the VIPs be selected for this first event?

**Reply by CaptainSplince** [reply]
Great question @okhaimie-dev.

VIPs will be invited journalists, Ecosystem partners, crypto influencers (think Vorhees, Vitalik or Sunny) and highly engaged community members. Generally, not all folks on a invited list will be available due to prior commitments, but the idea is to curate a list that I can review with @gosuri, the Overclock PR team, marketing team and potentially even Akash Insiders for signoff for invites.

**Comment by piber-dev** [comment]
hello @CaptainSplince, can you explain a little how the day could look like, roughly explaining how the main event could be like? i also wonder if you have any thoughts on if this event should only be one day, or if you think it could make sense to prolong it either virtually or not. i think a set of great workshops and technical explanations on the current state of akash could be included either in the main event or the days before.

**Reply by CaptainSplince** [reply]
Hey @piber-dev - I've drafted a run of show for the full scope of the event, but it's still very much under construction.

Right now the idea is:
- VIP dinner - Night before the summit (20+/- curated list of attendees) 
- Main event - multiple panels, keynotes, workshops over the course of a few hours (150 +/- open invite to anyone to attend)
- Networking Happy Hour after main event (150 +/- open invite to anyone to attend)

TBD on Livestream - depends on cost, however it will be recorded for review after event is over.

**Comment by KamuelBob** [comment]
Hey @CaptainSplince,

Love the idea. You noted that it would run before Consensus 2024 TBD, does that mean the likely location is Austin, Texas?

**Reply by CaptainSplince** [reply]
Hey @KamuelBob - Correct. 

We want it to coincide with Consensus 2024 so it provides to the greatest oppertunity for the most attendees as possible since they will be in town already for Consensus.

**Comment by ghost** [comment]
Akash presence in the whole crypto market is big, recognized and respected 

By having its own event Akash will solidify its image even more and officially establish as a Pilar within the crypto market 

More collaboration from similar projects in the space, plus attracting more talents are other benefits this prop could bring 

100% support

**Comment by Aihonaken** [comment]
i guess, this is exactly the right time to do these kinds of things. Market and 2024 narratives is a g2g for these programs! Love the idea, and make it as huge as possible !

**Reply by CaptainSplince** [reply]
Totally agree @Aihonaken.

**Comment by rodri-r** [comment]
Great proposal @CaptainSplince I love this idea and think that everything looks great. Just as it was mentioned before, I would just like to know where this event would take place at.  I think Barcelona would be an excellent choice for hosting this event.

**Reply by CaptainSplince** [reply]
Right now, because of timing and location of Consensus, the location will be in Austin. Having this event coincide with an already established event will help draw in the largest audience and attention. 🥇

**Comment by baktun14** [comment]
Full support of this initiative and looking forward to it!

**Comment by ArielLeza** [comment]
GREAT INITIATIVE 

if you see it relevant - I'd be happy to help with creating/presenting content, specifically in the realms of IT & Cloud (integrating Akash with existing cloud operations, how to become a provider a-z with the demo, best practices, and tools for operations, etc.).

if our goal is to strengthen the narrative of Akash as a valid solution for substituting some traditional cloud services, I think that demystifying these aspects could help to create a broader understanding of the solution and its applicability in real environments

**Reply by CaptainSplince** [reply]
Awesome.

Part of this event will include community support. Not only for workshops/talks but onsite event support. :-)

**Comment by anilmurty** [comment]
AKceleraTe! 🚀 🚀 🚀

**Comment by tgrecojs** [comment]
great write up @CaptainSplince! I am fully in support of this initiative (and I very much look forward to it taking place 🙂).

ATX is going to be AKTX by the time this is over! 🤠😎 🤠

**Comment by Corey-Code** [comment]
This is an incredible idea! Hosting Akash Accelerate could truly cement Akash's position in the decentralized AI space. The focus on ecosystem and technical achievements is spot-on. Excited to see how it unfolds and the kind of discussions it will spark!

**Comment by SophoclesSummer** [comment]
This proposal was approved on 3/15/2024 by a 98% majority.
https://www.mintscan.io/akash/proposals/250
